.gitignore

```diff
@@ -0,0 +1 @@
+auth.json.true
```

Comment:
this was just a dummy auth file that contained my actual login information, so i could swap it in for the scrubbed one and test locally. i'll remove this